tests/unit/test_analyzer_util_suite.py

```diff
@@ -136,3 +139,14 @@ def test_generate_scoped_temp_objects():
         )
         == f" CREATE    TABLE {temp_table_name}({temp_schema_name})"
     )
+
+
+def test_convert_value_to_sql_option():
+    assert convert_value_to_sql_option(True) == "True"
+    assert convert_value_to_sql_option("hello world") == "'hello world'"
+    assert convert_value_to_sql_option("'hello world'") == "'hello world'"
+    assert convert_value_to_sql_option("hello'world") == "'hello''world'"
+    assert convert_value_to_sql_option("''") == "''"
+    assert convert_value_to_sql_option("") == "''"
+    assert convert_value_to_sql_option(1) == "1"
+    assert convert_value_to_sql_option(None) == "None"
```

Comment:
Add this test, this will fail:
```
    assert convert_value_to_sql_option("'") == "''''"
```

Reply:
Added `assert convert_value_to_sql_option("'") == "''''"`.
Good point to have this test.